app/services/force/lease_up_service.rb

```diff
@@ -11,7 +11,7 @@ def lease_ups(listing_id)
         (SELECT Id FROM Listing_Lottery_Preference__c WHERE Listing__c = '#{listing_id}')
         AND Preference_Lottery_Rank__c != NULL
         ORDER BY Preference_Order__c, Preference_Lottery_Rank__c
-        ASC NULLS LAST LIMIT 50 OFFSET 1
+        ASC NULLS LAST LIMIT 500 OFFSET 1
```

Comment:
where does this limit come from?

Reply:
In implementing this feature, I couldn't see all the preference rows for an application to test the feature. I noticed that we weren’t querying all the lease up applications, they were just 50 at a time.

Removing the limit timed out the request. So I put it up to 500 for demoing purposes for the City, and creating https://www.pivotaltracker.com/story/show/159530254, which is to implement the same pagination on lease up applications as the non-lease-up applications page that we did in this story: https://www.pivotaltracker.com/story/show/157663604. A lot of the infrastructure to paginate was done so this will be smaller in scope :slightly_smiling_face:

We’ll do it in the next sprint.

Reply:
ah ok. should we add a fixme or a todo to make that a little clearer?

Reply:
Good idea, will add a ToDo!

Reply:
Done.